cap-ci/pipeline.yaml.tmpl

```diff
@@ -67,6 +68,30 @@ resources:
     secret_access_key: ((aws-secret-key))
     regexp: {{ $config.s3.regexp }}
     region_name: {{ $config.s3.region }}
+{{ end }}
```

Comment:
```suggestion
{{- end }}
```

Reply:
mmm, why? 
its {{ end}} at other places eg: https://github.com/SUSE/cap/blob/master/cap-ci/jobs/cf-acceptance-tests.tmpl#L37

Reply:
Because it's the pattern. It makes linting in the future easier. Since this is the first time I'm seeing this and I was not a reviewer of the other PRs, we can start by fixing this one.

For more context, we have a linter in KubeCF for catching this kind of pattern. Eventually, we can reuse to lint the pipelines in this repository.